crates/daphne-service-utils/src/test_route_types.rs

```diff
@@ -23,6 +26,47 @@ pub struct InternalTestVdaf {
     pub chunk_length: Option<String>,
 }
 
+impl From<VdafConfig> for InternalTestVdaf {
```

Comment:
What does "Internal" mean in this context?

Reply:
You came up with this name, [you tell me](https://github.com/cloudflare/daphne/pull/153) :eyes:

Reply:
It wasn't clear to me that this was related to one of the /internal/... endpoints. Perhaps just add some documentation.